spec/models/service_template_spec.rb

```diff
@@ -1020,6 +1021,29 @@
         end
       end
 
+      context "with parent request id" do
+        let(:arg2) { {:initiator => 'control', :submit_workflow => true, :parent_id => miq_automation_request.id} }
+
+        it "provisions a service template without errors" do
+          expect(resource_action_workflow).to receive(:validate_dialog).and_return([])
+          expect(resource_action_workflow).to receive(:make_request).and_return(miq_request)
+          expect(resource_action_workflow).to receive(:request_options=).with(
+            :initiator => 'control', :submit_workflow => true, :parent_id => miq_automation_request.id
+          )
+
+          expect(service_template.provision_request(user, arg1, arg2)).to eq(miq_request)
```

Comment:
I see an `expect()` checking the parent_id is passed in, can you add a spec showing the `parent` association is create properly?

Reply:
I'm assuming `miq_request.parent` is supposed to be an automate request here?

Reply:
>I see an expect() checking the parent_id is passed in, can you add a spec showing the parent association is create properly?

Added 
> I'm assuming miq_request.parent is supposed to be an automate request here?

Yes